cmd/consumer/main.go

```diff
@@ -2,19 +2,18 @@ package main
 
 import (
 	"context"
-	"time"
-
-	"github.com/trustwallet/blockatlas/services/notifier"
-
-	"github.com/trustwallet/blockatlas/config"
 	"github.com/trustwallet/blockatlas/services/subscriber"
 	"github.com/trustwallet/blockatlas/services/tokenindexer"
 	"github.com/trustwallet/blockatlas/services/tokensearcher"
+	"time"
+
+	"github.com/trustwallet/blockatlas/services/notifier"
 
 	log "github.com/sirupsen/logrus"
+	"github.com/trustwallet/blockatlas/config"
 	"github.com/trustwallet/blockatlas/db"
 	"github.com/trustwallet/blockatlas/internal"
-	"github.com/trustwallet/blockatlas/mq"
+	"github.com/trustwallet/blockatlas/new_mq"
```

Comment:
how about `mqueue`? sounds better than `new_`

Reply:
It will be just `mq`, `new_mq` will become `mq` as there is no need to keep the old one